promql/lex.go

```diff
@@ -127,6 +127,7 @@ const (
 	itemDIV
 	itemLAND
 	itemLOR
+	itemLUnless
```

Comment:
I'd love to drop the `L` suffic at some point. I find all uppercase hard to read and it's not following the go naming convention. There are many other `item*` constants using camel case as well, but I'm happy to change it to all uppercase if people think that makes more sense.